changelogs/fragments/iosxr-7.0.2-fix-integration-tests.yaml

```diff
@@ -0,0 +1,6 @@
+---
+major_changes:
```

Comment:
Does PR contain any breaking change while fixing the integration tests?
Adding new keys can be added as a minor change

Reply:
ack